Configuration/TypoScript/Structure/Gridelements.typoscript

```diff
@@ -0,0 +1,21 @@
+lib.gridelements.defaultGridSetup {
```

Comment:
Do we really need gridelements? If so, please add it to the dependancies in the composer.json.

Reply:
unfortunately yes, better working solutions will come in 10. I will move the composer Entry.